Cone_spanners_2/include/CGAL/Cone_spanners_2/Plane_scan_tree.h

```diff
@@ -182,14 +182,14 @@ class Plane_scan_tree {
 
     /* End Iterator */
     inline iterator end() {
-        static iterator res;
+        static const iterator res;
```

Comment:
For when do you plan that C++11 is the minimal version?    I do not ask for this particular case, but there are other issues/PR where this means that we have to  switch to thread local storage.